### PR TITLE
Remove $ from code block so that the copy feature can be used

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

I removed the $ from the code blocks so that people can just copy & paste code blocks

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Conflicts

File: documentation/docs/01_installation.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ https://www.rust-lang.org/tools/install
 ### 2. Compile
 
 ```sh
-$ git clone https://github.com/iotaledger/cli-wallet -b develop
-$ cd cli-wallet
-$ cargo build --profile production
+git clone git clone https://github.com/iotaledger/cli-wallet -b develop
```
Comment (Dr-Electron) [inline documentation/docs/01_installation.md]:
```suggestion
git clone https://github.com/iotaledger/cli-wallet -b develop
```

**Comment by aleksei-korolev** [comment]
These aren't code blocks, these are system shell commands that are supposed to be run by a normal system user (without elevated rights). This is what `$` indicates, and as a nice side effect it prevents people from copying the command blindly :trollface:

**Comment by thibault-martinez** [comment]
The presence of a `>` or a `$` in shell commands online is so widespread that I don't think it's an issue to have them tbh. But I don't really care if we remove them or not. You guys decide.

**Comment by Dr-Electron** [comment]
I know the meaning of `>`and `$`. But I prefer it being easy to copy.
The ideal solution would be to exclude those special cars from the copied content but I didn't find a way to do that yet.
I know other docs that do that, but as I said, didn't find a way in Docusaurus yet...

**Comment by Dr-Electron** [comment]
> These aren't code blocks, these are system shell commands that are supposed to be run by a normal system user (without elevated rights). This is what `$` indicates, and as a nice side effect it prevents people from copying the command blindly :trollface:

They are code blocks per definition tho :trollface: (The component is called code block).
And in an step-by-stepp guide it makes sense to me to just copy the content easily

**Comment by lucas-tortora** [comment]
> These aren't code blocks, these are system shell commands that are supposed to be run by a normal system user (without elevated rights). This is what `$` indicates, and as a nice side effect it prevents people from copying the command blindly :trollface:

UX-wise it's a bit irritating to have the extra `$` when you copy something, more so when it doesn't really add anything to the command. In my experience, it's not really going to stop from running, you're just going to get a `$: command not found` response on the first run, hit the up arrow, delete the `$`, and hit enter again.

**Comment by Dr-Electron** [comment]
Guess we can merge this then? 🤔 It's 2:1  :trollface:

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**